File: it.unibo.bls19Local/src/main/java/it/unibo/blsFramework/interfaces/IApplListener.java
package it.unibo.blsFramework.interfaces;

import it.unibo.bls.interfaces.IControlLed;
import it.unibo.bls.interfaces.IObserver;

public interface IApplListener extends IObserver{
    public void setControl(IAppLogic ctrl);
    public int getNumOfClicks();
}

File: it.unibo.bls19Local/src/main/kotlin/it/unibo/bls/kotlin/applLogic/IApplListener.kt
package it.unibo.bls.kotlin.applLogic

import it.unibo.bls.interfaces.IObserver
import it.unibo.blsFramework.interfaces.IAppLogic

interface IApplListener : IObserver {
    fun getNumOfClicks() : Int
    fun setControl(ctrl: IAppLogic)
}
